.github/workflows/tests.yml

```diff
@@ -533,24 +544,15 @@ jobs:
       with:
         prefix: windows
 
-    - name: Set up Elasticsearch and pdftotext
-      run: |
-        choco install xpdf-utils
-        choco install openjdk11
-        refreshenv
-        choco install elasticsearch --version=7.9.2
-        refreshenv
-        Get-Service elasticsearch-service-x64 | Start-Service
     - name: Install REST API and UI
       run: |
         pip install rest_api/
         pip install ui/
+
     - name: Run tests
-      env:
-        TOKENIZERS_PARALLELISM: 'false'
```

Comment:
I ask because our tests tend to be not very much unit-y, and often need fixing :sweat_smile: So I always check if there are unintended things going on.

Reply:
Now that tests are using mocks we can state they have to work without regard of the tokenizer settings